src/Compiler/Microsoft.CodeAnalysis.Razor/src/ComponentTagHelperDescriptorProvider.cs

```diff
@@ -354,6 +347,17 @@ private void CreateTypeParameterProperty(TagHelperDescriptorBuilder builder, ITy
             {
                 list.Add("unmanaged");
             }
+            else if (typeParameter.HasValueTypeConstraint)
```

Comment:
This `else if` intermixed with `if`s is pretty unreadable. Let's separate the `if`s with newlines to make that better.

Reply:
Thanks, updated: f1ca17ae8307972fed3432093c51fc20cf591f65